runelite-client/src/main/java/net/runelite/client/plugins/microbot/fletching/FletchingScript.java

```diff
@@ -148,19 +154,22 @@ private void bankItems(FletchingConfig config) {
     }
 
     private void fletch(FletchingConfig config) {
-        Rs2Inventory.combine(primaryItemToFletch, secondaryItemToFletch);
+        Rs2Inventory.combineClosest(primaryItemToFletch, secondaryItemToFletch);
         sleepUntil(() -> Rs2Widget.getWidget(17694736) != null);
         if (fletchingMode == FletchingMode.PROGRESSIVE) {
             keyPress(model.getFletchingItem().getOption(model.getFletchingMaterial(), fletchingMode));
+
         } else {
             keyPress(config.fletchingItem().getOption(config.fletchingMaterial(), fletchingMode));
         }
         sleepUntil(() -> Rs2Widget.getWidget(17694736) == null);
-        if (fletchingMode == FletchingMode.PROGRESSIVE) {
-            sleepUntil(() -> !Rs2Inventory.hasItemAmount(secondaryItemToFletch, model.getFletchingItem().getAmountRequired()) || hasLeveledUp, 60000);
-        } else {
-            sleepUntil(() -> !Rs2Inventory.hasItemAmount(secondaryItemToFletch, config.fletchingItem().getAmountRequired()) || hasLeveledUp, 60000);
-        }
+        Rs2Antiban.actionCooldown();
+        Rs2Antiban.takeMicroBreakByChance();
+//        if (fletchingMode == FletchingMode.PROGRESSIVE) {
```

Comment:
why is this piece of code in comment?

Reply:
Just a mistake left over from my testing, i have uncommented the code again in a new commit